doc/changelogs/CHANGELOG_V17.md

```diff
@@ -45,6 +45,26 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="17.9.1"></a>
+
+## 2022-05-31, Version 17.9.1 (Maintenance), @ruyadorno
```

Comment:
I think it should still specify  `(Current)` as for LTS releases in maintenance releases we still specify `(LTS)`. Only a nit/worth updating if you're amending for another reason, though.